app/locustio/jira/http_actions.py

```diff
@@ -261,7 +273,10 @@ def edit_issue_save_edit():
         r = locust.client.post(f'/secure/EditIssue.jspa?atl_token={locust.storage["atl_token"]}',
                                params=locust.storage['edit_issue_body'],
                                headers=TEXT_HEADERS, catch_response=True)
-        assert f'[{issue_key}]' in r.content.decode('utf-8')
+        content = r.content.decode('utf-8')
+        if not (f'[{issue_key}]' in content):
+            logger.error('Could not save edited page')
```

Comment:
need to add content like:
logger.error(f'User {user[0]} authentication failed: {content}')